src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -374,6 +390,39 @@ void RedisStoreClient::RedisScanner::OnScanCallback(
 
 int RedisStoreClient::GetNextJobID() { return redis_client_->GetNextJobID(); }
 
+Status RedisStoreClient::AsyncGetKeys(
+    const std::string &table_name,
+    const std::string &prefix,
+    std::function<void(std::vector<std::string>)> callback) {
+  std::string match_pattern = GenKeyRedisMatchPattern(table_name, prefix);
+  auto scanner = std::make_shared<RedisScanner>(redis_client_, table_name);
+  auto on_done = [table_name, callback, scanner](auto /* status*/, auto keys) {
+    std::vector<std::string> result;
```

Comment:
If status is failed, keys are just empty?

Reply:
turns out the status is always OK; or it will crash.